### ISSUE TITLE
Plugin not working with new Woocommerce Checkout Block

### ISSUE THREAD

**Issue by jimon07** [Cardlink-SA/cardlink-payment-gateway-woocommerce#6]
When added the new Woocommerce Checkout Block as the checkout page the payment method is not the as an option. Is this a possible fix because the new page is much better.

**Comment by spithash** [comment]
I am on Version 8.7.0 and it works as expected, are you sure it's not something else that's blocking it?

**Comment by jimon07** [comment]
Yes when i create a test page with the checkout block the card payment option is missing. 
As shown on the picture below: 
https://prnt.sc/CdYLpShs0Z7N

On the old checkout page it is shown correctly: 
https://prnt.sc/bB6T7GJOQoyx
